cmd/operator-sdk/alpha/scorecard/cmd.go

```diff
@@ -33,7 +34,15 @@ func NewCmd() *cobra.Command {
 		Long:   `Has flags to configure dsl, bundle, and selector.`,
 		Hidden: true,
 		Run: func(cmd *cobra.Command, args []string) {
-			log.Info("TODO")
+			scorecardFlags := scorecard.ScorecardFlags{
+				Config:   config,
+				Bundle:   bundle,
+				Selector: selector,
+				ListAll:  listAll,
+			}
+			if err := scorecard.RunTests(scorecardFlags); err != nil {
```

Comment:
It is possible to validate the options as a first step? This would be so that if we fail to validate, we can return an error instead of doing `log.Fatal`, which will result in cobra printing the usage text for the command.

Reply:
I updated this in the latest commit based on your suggestion.